pull_request_template.md

```diff
@@ -0,0 +1,118 @@
+## PR Writing Checklist
+
+If your PR involves adding or updating writing content,
+please check the following items before submitting your PR for review:
+
+- [ ] **Avoid [inline links].
+  Instead, use [reference links] with a list of
+  [link reference definitions] at the bottom of your page.**
+
+### Example:
+
+Not recommended:
+
+```md
+For more information, consult
+[reference links](https://github.github.com/gfm#reference-links)
+for [GitHub Flavored Markdown](https://github.github.com/gfm).
+
+```
+
+Recommended:
+
+```md
+For more information, consult [reference links] for [GitHub Flavored Markdown].
+
+(Include this list of link reference definitions at the bottom of the markdown file)
+
+[reference links]: https://github.github.com/gfm/#reference-link
+[GitHub Flavored Markdown]: https://github.github.com/gfm/
+```
+
+- [ ] **Avoid duplicating a referenced link definition by using
+  [link labels] after [reference links].**
+
+### Example:
+
+Not recommended:
+```md
+[Foo] and [bar] link to the same URL.
+
+[Foo]: https://google.com
+[bar]: https://google.com
+```
+
+Recommended:
+```md
+[Foo][google] and [bar][google] link to the same URL.
```

Comment:
Actually, I think this is a bad example. In this case I'd do:

[Foo and bar]: link

Reply:
`Actually, I think this is a bad example. In this case I'd do:`

^ Did you mean to finish this thought? The comment appears to end here.

Reply:
No, there was more, but it was interpreted rather than displayed. I've marked it as code so it shows up now.

Reply:
Combining these into `[foo and bar]` would fail to demonstrate how to use the label, right?

Reply:
Or are you recommending that we find an example that makes the need for a label for compelling?